docs/classad-attributes/machine-classad-attributes.rst

```diff
@@ -40,6 +40,14 @@ Machine ClassAd Attributes
     ``"X86_64"``
         AMD/Intel 64-bit X86
 
+:classad-attribute-def:`AvgTransferInputMB`
+    The average number of megabytes transferred to each job to build the job
+    execution sandbox.
+
+:classad-attribute-def:`AvgTransferOutputMB`
+    The average number of Megabytes transferred from each job from the execution
+    sandbox after the job has ended.
```

Comment:
```suggestion
    The average number of megabytes transferred from each job from the execution
    sandbox after the job has ended.
```

Reply:
Also, consider specifying here and in the `TotalTransfer` attributes that the average and sum are computed over the lifetime of the startd.